microdaq/macros/microdaq_macros/mdaq_ao_scan_init.sci

```diff
@@ -64,6 +64,7 @@ function  mdaq_ao_scan_init(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
     end
 
     if scan_time < 0 then
```

Comment:
To make sense for this warning line below,  the condition should be like this: 
if scan_time < 0 & scan_time <> -1 then